skyportal/handlers/api/candidate.py

```diff
@@ -394,6 +417,18 @@ def get(self, obj_id=None):
             )
             .outerjoin(Annotation)
         )  # Join in annotations info for sort/filter
+        if classifications is not None:
+            if isinstance(classifications, str) and "," in classifications:
+                classifications = classifications.split(",")
```

Comment:
```suggestion
                classifications = [c.strip() for c in classifications.split(",")]
```
In case someone provides something like `"classA, classB"`

Reply:
done